packages/output/src/browser/output-contribution.ts

```diff
@@ -16,7 +16,16 @@
 
 import { injectable } from 'inversify';
 import { AbstractViewContribution } from '@theia/core/lib/browser/shell/view-contribution';
+import { Widget } from '@theia/core/lib/browser';
 import { OUTPUT_WIDGET_KIND, OutputWidget } from './output-widget';
+import { Command, CommandRegistry } from '@theia/core/lib/common';
+
+export namespace OutputCommands {
+    export const CLEAR_ALL_TOOLBAR: Command = {
```

Comment:
I wonder, should we also include the **Clear Output** quick-open command like in vscode?
All it'd do is simply call the new **`clear()`** method.

Reply:
Also we could add to the keybindings.  Perhaps this is a discussion for another PR.  I have changed the id to output:clear as that is consistent with terminal:clear, though there does not seem to be a convention for ids.

Reply:
I'm fine with the `id`, all I was suggesting is to provide a `label` to the **`CLEAR_ALL_TOOLBAR`** (`Clear Output`) so that it appears as a quick-command like in vscode.

Reply:
OK, I did not realize all you were suggesting was adding the label.  I've done that, and a category.

I've created https://github.com/theia-ide/theia/issues/5298 to cover the additional work that will be needed to register the command as a Quick Open action (i.e. implementations for QuickOpenContribution, QuickOpenHandler etc).